### PR TITLE
Switch to the new agreed Semantic Versioning scheme

### PR DESCRIPTION
**Pull request by adamretter**
When building distributions from a `SNAPSHOT` you will get artifacts named like:

```
eXist-db-setup-3.1.0-SNAPSHOT+20170307003600.jar
eXist-db-3.1.0-SNAPSHOT+20170307003600.dmg
eXist-db-3.1.0-SNAPSHOT+20170307003600.war
eXist-db-3.1.0-SNAPSHOT+20170307003600.zip
eXist-db-3.1.0-SNAPSHOT+20170307003600.tgz
```

When building distributions from an `RC1` you will get artifacts named like:
```
eXist-db-setup-3.1.0-RC1.jar
eXist-db-3.1.0-RC1.dmg
eXist-db-3.1.0-RC1.war
eXist-db-3.1.0-RC1.zip
eXist-db-3.1.0-RC1.tgz
```

When building distributions from a release tag you will get artifacts named like:
```
eXist-db-setup-3.1.0.jar
eXist-db-3.1.0.dmg
eXist-db-3.1.0.war
eXist-db-3.1.0.zip
eXist-db-3.1.0.tgz
```

* I improved the attributes in the Jar manifest files so that they follow the standard naming conventions of the Java spec.

* I also removed support entirely for Launch4j Windows Executables as in todays community teleconference it was revealed that these are no longer supported and were not previously shipped for eXist 3.0.

Closes https://github.com/eXist-db/exist/pull/1276

### CODE REVIEW

**Comment by adamretter** [comment]
I am afraid I haven't yet had the time to send the PR for the Versioning Scheme documentation, I will do so very soon, but it is not actually required for the release itself.

**Comment by joewiz** [comment]
```
prepare:
     [echo] --------------------------------------------
     [echo] eXist 3.1.0-SNAPSHOT+20170306011700 build
     [echo] --------------------------------------------
     [echo] Git commit ID: pr_1327-f0d3647ec
```

Whoo hoo!

**Comment by joewiz** [comment]
```
download-xar:
    [fetch] Getting: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] To: /var/folders/g7/j_gtgs2j7fgcnn2w46lrsbj40000gn/T/FetchTask3503693117876822323tmp
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Can't get http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT to /var/folders/g7/j_gtgs2j7fgcnn2w46lrsbj40000gn/T/FetchTask3503693117876822323tmp
    [fetch] Can't get http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT to /var/folders/g7/j_gtgs2j7fgcnn2w46lrsbj40000gn/T/FetchTask3503693117876822323tmp

download:

download-xar:
    [fetch] Getting: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] To: /var/folders/g7/j_gtgs2j7fgcnn2w46lrsbj40000gn/T/FetchTask548142173420182616tmp
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Error opening connection java.io.IOException: Server returned HTTP response code: 400 for URL: http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT
    [fetch] Can't get http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT to /var/folders/g7/j_gtgs2j7fgcnn2w46lrsbj40000gn/T/FetchTask548142173420182616tmp
    [fetch] Can't get http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=monex&zip=yes&processor=3.1.0-SNAPSHOT to /var/folders/g7/j_gtgs2j7fgcnn2w46lrsbj40000gn/T/FetchTask548142173420182616tmp
```

Boo hoo! :(

**Comment by joewiz** [comment]
The URL http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=eXide&zip=yes&processor=3.1.0-SNAPSHOT yields:

```
<exception>
  <path>/db/apps/public-repo/modules/find.xql</path>
  <message>exerr:ERROR Type mismatch: the actual return type does not match the type declared in the signature of function: scanrepo:version-to-number(xs:string) xs:int. Required type: xs:int, got xs:double. [at line 20, column 53, source: /db/apps/public-repo/modules/scan.xql]
In function:
	scanrepo:version-to-number(xs:string) [27:12:/db/apps/public-repo/modules/scan.xql]
	(item()*, item()*) [12:62:/db/apps/public-repo/modules/scan.xql]
	scanrepo:check-version(xs:string, xs:string, function) [12:9:/db/apps/public-repo/modules/scan.xql]
	scanrepo:is-newer-or-same(xs:string, xs:string?) [60:21:/db/apps/public-repo/modules/scan.xql]
	scanrepo:find-newest(element()*, element()?, xs:string?) [197:13:/db/apps/public-repo/modules/scan.xql]
	app:find-version(element()*, xs:string, xs:string?, xs:string?, xs:string?, xs:string?) [20:14:/db/apps/public-repo/modules/app.xql]</message>
</exception>
```

The error points to https://github.com/eXist-db/public-xar-repo/blob/master/modules/scan.xql#L20-L24 - so we'll await Wolfgang's patch to the public-xar-repo.

**Comment by joewiz** [comment]
Interestingly, http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=dashboard&zip=yes&processor=3.1.0-SNAPSHOT and http://demo.exist-db.org/exist/apps/public-repo/pkg.zip?abbrev=shared&zip=yes&processor=3.1.0-SNAPSHOT downloaded without error.

**Comment by adamretter** [comment]
@joewiz interesting mix of results. I will try and investigate tomorrow AM as well

**Comment by joewiz** [comment]
A few more observations. I ran `./build.sh rebuild` and while the build completed, I searched for portions of the build console log that included the word `error`, and found 2 - one involving betterform, the other involving contentextraction, and both involving ivy:

**extension-betterform**

> [ivy:resolve] 	unknown resolver default
> [ivy:resolve] 	unknown resolver main

full log:

```
extension-betterform:

resolve:
[ivy:resolve] :: Apache Ivy 2.4.0 - 20141213170938 :: http://ant.apache.org/ivy/ ::
[ivy:resolve] :: loading settings :: file = /Users/joe/workspace/exist/extensions/betterform/ivysettings.xml
[ivy:resolve] :: resolving dependencies :: org.exist#betterfrom;working@choski-pro-13.local
[ivy:resolve] 	confs: [default]
[ivy:resolve] 	found org.jgroups#jgroups;3.4.1.Final in central
[ivy:resolve] 	found org.jboss.spec.javax.transaction#jboss-transaction-api_1.1_spec;1.0.1.Final in central
[ivy:resolve] 	found org.jboss.marshalling#jboss-marshalling;1.3.18.GA in central
[ivy:resolve] 	found org.jboss.modules#jboss-modules;1.1.0.Beta1 in central
[ivy:resolve] 	found org.jboss.marshalling#jboss-marshalling-river;1.3.18.GA in central
[ivy:resolve] 	found org.jboss.logging#jboss-logging;3.1.2.GA in central
[ivy:resolve] 	found org.jboss.logmanager#jboss-logmanager;1.2.0.GA in central
[ivy:resolve] 	found org.infinispan#infinispan-commons;6.0.0.Final in central
[ivy:resolve] 	found org.jboss.logging#jboss-logging-processor;1.1.0.Final in central
[ivy:resolve] 	found org.jboss.jdeparser#jdeparser;1.0.0.Final in central
[ivy:resolve] 	found net.jcip#jcip-annotations;1.0 in chain
[ivy:resolve] 	found org.infinispan#infinispan-core;6.0.0.Final in central
[ivy:resolve] 	found de.betterform#betterform-exist;5.1-SNAPSHOT-20160615 in exist
[ivy:resolve] :: resolution report :: resolve 871ms :: artifacts dl 33ms
	---------------------------------------------------------------------
	|                  |            modules            ||   artifacts   |
	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
	---------------------------------------------------------------------
	|      default     |   13  |   0   |   0   |   0   ||   13  |   0   |
	---------------------------------------------------------------------
[ivy:resolve] 
[ivy:resolve] :: problems summary ::
[ivy:resolve] :::: ERRORS
[ivy:resolve] 	unknown resolver default
[ivy:resolve] 	unknown resolver main
[ivy:resolve] 
[ivy:resolve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
[ivy:retrieve] :: retrieving :: org.exist#betterfrom
[ivy:retrieve] 	confs: [default]
[ivy:retrieve] 	13 artifacts copied, 0 already retrieved (18875kB/101ms)
```

**download-3rd-party > contentextraction**

> [ivy:retrieve] 	unknown resolver chain
> [ivy:retrieve] 	unknown resolver chain
> [ivy:retrieve] 	unknown resolver chain

full log:

```
download-3rd-party:
     [echo] Retrieving 3rd party jar files 'contentextraction'
[ivy:retrieve] :: loading settings :: url = jar:file:/Users/joe/workspace/exist/tools/ant/lib/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[ivy:retrieve] :: resolving dependencies :: org.exist#tika;working@choski-pro-13.local
[ivy:retrieve] 	confs: [default]
[ivy:retrieve] 	found org.apache.tika#tika-parsers;1.12 in public
[ivy:retrieve] 	found org.apache.tika#tika-core;1.12 in public
[ivy:retrieve] 	found org.gagravarr#vorbis-java-tika;0.6 in public
[ivy:retrieve] 	found com.healthmarketscience.jackcess#jackcess;2.1.2 in public
[ivy:retrieve] 	found commons-lang#commons-lang;2.6 in public
[ivy:retrieve] 	found com.healthmarketscience.jackcess#jackcess-encrypt;2.1.1 in public
[ivy:retrieve] 	found org.bouncycastle#bcprov-jdk15on;1.52 in public
[ivy:retrieve] 	found net.sourceforge.jmatio#jmatio;1.0 in public
[ivy:retrieve] 	found org.apache.james#apache-mime4j-core;0.7.2 in public
[ivy:retrieve] 	found org.apache.james#apache-mime4j-dom;0.7.2 in public
[ivy:retrieve] 	found org.tukaani#xz;1.5 in public
[ivy:retrieve] 	found org.apache.pdfbox#pdfbox;1.8.10 in public
[ivy:retrieve] 	found org.apache.pdfbox#fontbox;1.8.10 in public
[ivy:retrieve] 	found org.apache.pdfbox#jempbox;1.8.10 in public
[ivy:retrieve] 	found org.bouncycastle#bcmail-jdk15on;1.52 in public
[ivy:retrieve] 	found org.bouncycastle#bcpkix-jdk15on;1.52 in public
[ivy:retrieve] 	found org.apache.poi#poi;3.13 in public
[ivy:retrieve] 	found org.apache.poi#poi-scratchpad;3.13 in public
[ivy:retrieve] 	found org.apache.poi#poi-ooxml;3.13 in public
[ivy:retrieve] 	found org.apache.poi#poi-ooxml-schemas;3.13 in public
[ivy:retrieve] 	found org.apache.xmlbeans#xmlbeans;2.6.0 in public
[ivy:retrieve] 	found org.ccil.cowan.tagsoup#tagsoup;1.2.1 in public
[ivy:retrieve] 	found com.googlecode.mp4parser#isoparser;1.0.2 in public
[ivy:retrieve] 	found com.drewnoakes#metadata-extractor;2.8.0 in public
[ivy:retrieve] 	found com.adobe.xmp#xmpcore;5.1.2 in public
[ivy:retrieve] 	found de.l3s.boilerpipe#boilerpipe;1.1.0 in public
[ivy:retrieve] 	found com.rometools#rome;1.5.1 in public
[ivy:retrieve] 	found com.rometools#rome-utils;1.5.1 in public
[ivy:retrieve] 	found org.gagravarr#vorbis-java-core;0.6 in public
[ivy:retrieve] 	found com.googlecode.juniversalchardet#juniversalchardet;1.0.3 in public
[ivy:retrieve] 	found org.codelibs#jhighlight;1.0.2 in public
[ivy:retrieve] 	found com.pff#java-libpst;0.8.1 in public
[ivy:retrieve] 	found com.github.junrar#junrar;0.7 in public
[ivy:retrieve] 	found org.apache.commons#commons-vfs2;2.0 in public
[ivy:retrieve] 	found org.apache.opennlp#opennlp-tools;1.5.3 in public
[ivy:retrieve] 	found org.apache.opennlp#opennlp-maxent;3.0.3 in public
[ivy:retrieve] 	found net.sf.jwordnet#jwnl;1.3.3 in public
[ivy:retrieve] 	found org.apache.commons#commons-exec;1.3 in default
[ivy:retrieve] 	found com.googlecode.json-simple#json-simple;1.1.1 in public
[ivy:retrieve] 	found org.json#json;20140107 in public
[ivy:retrieve] 	found com.google.code.gson#gson;2.2.4 in public
[ivy:retrieve] 	found edu.ucar#netcdf4;4.5.5 in public
[ivy:retrieve] 	found net.jcip#jcip-annotations;1.0 in default
[ivy:retrieve] 	found net.java.dev.jna#jna;4.1.0 in default
[ivy:retrieve] 	found edu.ucar#grib;4.5.5 in public
[ivy:retrieve] 	found com.google.protobuf#protobuf-java;2.5.0 in public
[ivy:retrieve] 	found org.jdom#jdom2;2.0.4 in public
[ivy:retrieve] 	found org.jsoup#jsoup;1.7.2 in public
[ivy:retrieve] 	found edu.ucar#jj2000;5.2 in public
[ivy:retrieve] 	found org.itadaki#bzip2;0.9.1 in public
[ivy:retrieve] 	found edu.ucar#cdm;4.5.5 in public
[ivy:retrieve] 	found edu.ucar#udunits;4.5.5 in public
[ivy:retrieve] 	found joda-time#joda-time;2.2 in public
[ivy:retrieve] 	found edu.ucar#httpservices;4.5.5 in public
[ivy:retrieve] 	found com.google.guava#guava;17.0 in public
[ivy:retrieve] 	found com.beust#jcommander;1.35 in public
[ivy:retrieve] 	found org.apache.commons#commons-csv;1.0 in public
[ivy:retrieve] 	found org.apache.sis.core#sis-utility;0.5 in public
[ivy:retrieve] 	found org.opengis#geoapi;3.0.0 in public
[ivy:retrieve] 	found javax.measure#jsr-275;0.9.3 in public
[ivy:retrieve] 	found org.apache.sis.storage#sis-netcdf;0.5 in public
[ivy:retrieve] 	found org.apache.sis.storage#sis-storage;0.5 in public
[ivy:retrieve] 	found org.apache.sis.core#sis-metadata;0.5 in public
[ivy:retrieve] 	found org.apache.sis.core#sis-referencing;0.5 in public
[ivy:retrieve] 	found org.apache.ctakes#ctakes-core;3.2.2 in public
[ivy:retrieve] 	found org.apache.ctakes#ctakes-core-res;3.2.2 in public
[ivy:retrieve] 	found org.apache.ctakes#ctakes-type-system;3.2.2 in public
[ivy:retrieve] 	found org.apache.uima#uimaj-core;2.5.0 in public
[ivy:retrieve] 	found org.springframework#spring-context;3.1.2.RELEASE in public
[ivy:retrieve] 	found org.springframework#spring-aop;3.1.2.RELEASE in public
[ivy:retrieve] 	found aopalliance#aopalliance;1.0 in public
[ivy:retrieve] 	found org.springframework#spring-asm;3.1.2.RELEASE in public
[ivy:retrieve] 	found org.springframework#spring-beans;3.1.2.RELEASE in public
[ivy:retrieve] 	found org.springframework#spring-core;3.1.2.RELEASE in public
[ivy:retrieve] 	found org.springframework#spring-expression;3.1.2.RELEASE in public
[ivy:retrieve] 	found tw.edu.ntu.csie#libsvm;3.1 in public
[ivy:retrieve] 	found org.apache.uima#uimafit-core;2.1.0 in public
[ivy:retrieve] 	found org.apache.ctakes#ctakes-utils;3.2.2 in public
[ivy:retrieve] 	found org.apache.uima#uimaj-tools;2.6.0 in public
[ivy:retrieve] 	found org.apache.uima#uimaj-cpe;2.6.0 in public
[ivy:retrieve] 	found org.apache.uima#uimaj-adapter-vinci;2.6.0 in public
[ivy:retrieve] 	found org.apache.uima#jVinci;2.6.0 in public
[ivy:retrieve] 	found org.cleartk#cleartk-util;2.0.0 in public
[ivy:retrieve] 	found edu.mit.findstruct#findstructapi;0.0.1 in public
[ivy:retrieve] 	found com.googlecode.armbrust-file-utils#sqlwrapper;0.0.1 in public
[ivy:retrieve] 	found net.sourceforge.openai#openaifsm;0.0.1 in public
[ivy:retrieve] 	found org.apache.uima#uimaj-examples;2.4.0 in public
[ivy:retrieve] 	found org.apache.uima#uimaj-document-annotation;2.5.0 in public
[ivy:retrieve] :: resolution report :: resolve 2238ms :: artifacts dl 76ms
[ivy:retrieve] 	:: evicted modules:
[ivy:retrieve] 	org.apache.tika#tika-core;1.5 by [org.apache.tika#tika-core;1.12] in [default]
[ivy:retrieve] 	com.healthmarketscience.jackcess#jackcess;2.1.0 by [com.healthmarketscience.jackcess#jackcess;2.1.2] in [default]
[ivy:retrieve] 	commons-lang#commons-lang;2.4 by [commons-lang#commons-lang;2.6] in [default]
[ivy:retrieve] 	org.jdom#jdom2;2.0.3 by [org.jdom#jdom2;2.0.4] in [default]
[ivy:retrieve] 	com.google.guava#guava;10.0 by [com.google.guava#guava;17.0] in [default]
[ivy:retrieve] 	org.apache.uima#uimaj-cpe;2.4.0 by [org.apache.uima#uimaj-cpe;2.6.0] in [default]
[ivy:retrieve] 	org.apache.uima#uimaj-tools;2.4.0 by [org.apache.uima#uimaj-tools;2.6.0] in [default]
	---------------------------------------------------------------------
	|                  |            modules            ||   artifacts   |
	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
	---------------------------------------------------------------------
	|      default     |   95  |   0   |   0   |   7   ||   88  |   0   |
	---------------------------------------------------------------------
[ivy:retrieve] 
[ivy:retrieve] :: problems summary ::
[ivy:retrieve] :::: ERRORS
[ivy:retrieve] 	unknown resolver chain
[ivy:retrieve] 	unknown resolver chain
[ivy:retrieve] 	unknown resolver chain
```

**Comment by joewiz** [comment]
When manually moving eXide-2.3.2.xar and monex-0.9.6.xar into the autodeploy directory and starting up after a `./build.sh clean-default-data-dir`, I get these errors in expath-repo.log:

```
2017-03-06 22:29:55,718 [Thread-3] WARN  (AutoDeploymentTrigger.java [execute]:95) - Exception during deployment of app eXide-2.3.2.xar: Package requires eXistdb version > 3.0.0 < 3.2.0. Installed version is 1.0 
org.expath.pkg.repo.PackageException: Package requires eXistdb version > 3.0.0 < 3.2.0. Installed version is 1.0
	at org.exist.repo.Deployment.checkProcessorVersion(Deployment.java:292) ~[exist.jar:3.1.0-SNAPSHOT]
	at org.exist.repo.Deployment.installAndDeploy(Deployment.java:217) ~[exist.jar:3.1.0-SNAPSHOT]
	at org.exist.repo.AutoDeploymentTrigger.execute(AutoDeploymentTrigger.java:93) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.StartupTriggersManager.startPreMultiUserSystem(StartupTriggersManager.java:58) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPoolServicesManager.startPreMultiUserSystemServices(BrokerPoolServicesManager.java:190) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPool._initialize(BrokerPool.java:631) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPool.initialize(BrokerPool.java:415) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPools.configure(BrokerPools.java:181) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPools.configure(BrokerPools.java:110) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.jetty.JettyStart.run(JettyStart.java:164) [exist-optional.jar:3.1.0-SNAPSHOT]
	at org.exist.launcher.Launcher$2.run(Launcher.java:177) [exist.jar:3.1.0-SNAPSHOT]
2017-03-06 22:29:55,730 [Thread-3] INFO  (Deployment.java [installAndDeploy]:219) - Package http://exist-db.org/apps/monex depends on http://exist-db.org/apps/shared 
2017-03-06 22:29:55,732 [Thread-3] DEBUG (Deployment.java [installAndDeploy]:222) - Package http://exist-db.org/apps/shared already installed 
2017-03-06 22:29:55,734 [Thread-3] DEBUG (Deployment.java [installAndDeploy]:243) - Package http://exist-db.org/apps/shared already installed 
2017-03-06 22:29:55,734 [Thread-3] WARN  (AutoDeploymentTrigger.java [execute]:95) - Exception during deployment of app monex-0.9.6.xar: Package requires eXistdb version > 3.0.2 < 3.1.0. Installed version is 1.0 
org.expath.pkg.repo.PackageException: Package requires eXistdb version > 3.0.2 < 3.1.0. Installed version is 1.0
	at org.exist.repo.Deployment.checkProcessorVersion(Deployment.java:292) ~[exist.jar:3.1.0-SNAPSHOT]
	at org.exist.repo.Deployment.installAndDeploy(Deployment.java:217) ~[exist.jar:3.1.0-SNAPSHOT]
	at org.exist.repo.AutoDeploymentTrigger.execute(AutoDeploymentTrigger.java:93) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.StartupTriggersManager.startPreMultiUserSystem(StartupTriggersManager.java:58) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPoolServicesManager.startPreMultiUserSystemServices(BrokerPoolServicesManager.java:190) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPool._initialize(BrokerPool.java:631) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPool.initialize(BrokerPool.java:415) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPools.configure(BrokerPools.java:181) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.storage.BrokerPools.configure(BrokerPools.java:110) [exist.jar:3.1.0-SNAPSHOT]
	at org.exist.jetty.JettyStart.run(JettyStart.java:164) [exist-optional.jar:3.1.0-SNAPSHOT]
	at org.exist.launcher.Launcher$2.run(Launcher.java:177) [exist.jar:3.1.0-SNAPSHOT]
```

I think the reports above are all the ones Wolfgang anticipated - so nothing new here. I'm just pasting these in hoping they help jump start the process tomorrow.

File: build/scripts/build-impl.xml [hunk 1]
```diff
@@ -344,11 +343,26 @@
...
         <jar basedir="${build.classes}" compress="true" jarfile="exist.jar">
             <manifest>
-                <attribute name="Project-Name" value="${project.name}"/>
-                <attribute name="Project-Version" value="${project.version}"/>
-                <attribute name="Project-Build" value="${DSTAMP}"/>
-                <attribute name="Project-CodeName" value="${project.codename}"/>
-                <attribute name="Git-commit" value="${git.commit}"/>
+                <attribute name="Implementation-Title" value="${project.name} Core"/>
+                <attribute name="Implementation-Version" value="${project.version}"/>
+                <attribute name="Implementation-Vendor" value="The eXist Project"/>
```
Comment (shabanovd) [inline build/scripts/build-impl.xml]: eXist-db?

**Comment by adamretter** [comment]
@joewiz I think those Ivy errors were likely there before. We should check...

File: build/scripts/build-impl.xml (hunk 1 above)
Comment (adamretter) [inline build/scripts/build-impl.xml]: I took the `Implementation-Vendor` name from the name that we use in the LGPL header, as that is as close to anything that we have which actually defines an name for the open source organisation.

**Comment by adamretter** [comment]
@joewiz Looks like we will indeed need to patch the EXPath version check; I think perhaps @wolfgangmm already suggested in the call yesterday that he would look at that.

File: build/scripts/build-impl.xml (hunk 1 above)
Comment (shabanovd) [inline build/scripts/build-impl.xml]: The naming issue was raised by @joewiz some time ago ("db or not db: The many names of eXist") and agreement was to use `eXist-db`, right?

**Comment by shabanovd** [comment]
@adamretter that is not EXPath issue because it clearly see eXist version 1.0 - "Package requires eXistdb version > 3.0.2 < 3.1.0. Installed version is **1.0**". That'll be EXPath issue only if we see message like "Package requires eXistdb version > 3.0.2 < 3.1.0. Installed version is **3.1.0-SNAPSHOT**".

**Comment by wolfgangmm** [comment]
There are two issues here:

1. the system property product-semver does no longer exist, so the check code falls back to "1.0". This needs to be replaced by product-version.
2. the expath pkg library expects a version with 3 numeric parts

I'll try to address both.

File: build/scripts/build-impl.xml (hunk 1 above)
Comment (joewiz) [inline build/scripts/build-impl.xml]: I would say the original post's characterizations held up through that thread. To summarize: the community uses both (actually, more than two!) variants in different circumstances and according to individual preferences and conventions.

**Comment by wolfgangmm** [comment]
Ok, so in combination with #1334 everything should work again.

**Comment by joewiz** [comment]
I can confirm that the `download-xar` portion of the build works perfectly now with #1334 and presumably also the public-repo on exist-db.org having been patched with https://github.com/eXist-db/public-xar-repo/commit/d0bc5a7c71b0c3d2e5890bd2252d40ab68962c02. Startup also completes without any errors in `expath-repo.log`. 

Thanks @adamretter and @wolfgangmm!

**Comment by ljo** [comment]
Yes, this was all expected as of discussions last week. Great that it now seems to be resolved for default apps! Hurray.

**Comment by StephanMa** [comment]
Are there some news concerning the ivy - betterFORM Problem?

**Pull request merged**